assets/src/scripts/components/dailyValueHydrograph/legend.js

```diff
@@ -0,0 +1,94 @@
+// functions to facilitate DV legend creation for a d3 plot
+import {createStructuredSelector} from 'reselect';
+
+import {getLayout} from './selectors/layout';
+import {getLegendMarkerRows} from './selectors/legend-data';
+
+import config from '../../config';
+import { mediaQuery } from '../../utils';
+import {link} from '../../lib/d3-redux';
+
+
+/**
+ * Create a simple legend
+ *
+ * @param {Object} div - d3 selector where legend should be created
+ * @param {Object} legendMarkerRows - Array of rows. Each row should be an array of legend markers.
+ * @param {Object} layout - width and height of svg.
+ */
+export const drawSimpleLegend = function(div, {legendMarkerRows, layout}) {
```

Comment:
This drawSimpleLegend is the same as the one in hydrograph/legend.js. Let's move this into a new module in d3-rendering (probably called legend.js). Then this module would export the single function drawTimeSeriesLegend. You should change hydrograph/legend.js to remove drawSimpleLegend and instead import it.